src/utils/bytes.js

```diff
@@ -32,3 +32,28 @@ export function stringToBytes(str) {
   }
   return bytes;
 };
+
+/**
+ * Converts a 8-bit bytes array into a string
+ * @param {!Uint8Array} bytes
+ * @return {string}
+ */
+export function bytesToString(bytes) {
+  return String.fromCharCode.apply(String, bytes);
+};
+
+/**
+ * Generate a random bytes array with specific length
+ * @param {!number} length
+ * @return {!Uint8Array}
+ */
+export function getRandomBytesArray(length) {
```

Comment:
You are totally correct. But here we are using `Math.random()` only once.

Actually, the thing we can really criticize here is that the function doesn't support big `length`.